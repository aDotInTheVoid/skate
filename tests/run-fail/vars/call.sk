fn bar() {

}

fn main() {
    bar(x, y);
}