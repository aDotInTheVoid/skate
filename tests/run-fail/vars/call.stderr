error: Couldn't find variable `x` in scope
  ┌─ $DD/tests/run-fail/vars/call.sk:6:9
  │
6 │     bar(x, y);
  │         ^
  │
  = Variables in scope: []

